src/meltano/api/controllers/orchestrations.py

```diff
@@ -473,34 +475,50 @@ async def test_extractor():
 def get_pipeline_schedules():
     """Endpoint for getting the pipeline schedules.
 
+    Note that unless the ff ENABLE_API_SCHEDULED_JOB_LIST is enabled this endpoint will filter out scheduled jobs.
+
     Returns:
         JSON containing the pipline schedules.
     """
     project = Project.find()
     schedule_service = ScheduleService(project)
     schedules = list(map(dict, schedule_service.schedules()))
+
+    jobs_in_list = False
+    with ProjectSettingsService(project).feature_flag(
+        FeatureFlags.ENABLE_API_SCHEDULED_JOB_LIST, raise_error=False
+    ) as allow:
+        if allow:
+            jobs_in_list = True
+
```

Comment:
@aaronsteers I wrapped that short-circuit to the API listing that we talked about in a feature flag. Mainly to make sure that we don't "forget" about it 😁